features/step_definitions/employee_creates_and_changes_objectives_steps.rb

```diff
@@ -98,3 +101,11 @@
   ]
   expect(@report.smart).to match_array(expected_smart)
 end
+
+When(/^I view dashboard$/) do
+  dashboard.load
+end
+
+Then(/^I cannot create a new report$/) do
+  expect(@dashboard.has_create_new_report?).to eq(false)
```

Comment:
you can keep the original format, but you have to use the `has_no...` Capybara selector, otherwise it's not reliable. So:

```ruby
expect(@dashboard.has_no_create_new_report?).to eq(true)
```